### ISSUE TITLE
future function

### ISSUE THREAD

**Issue by noke8868** [hennande/Temporal_Relational_Stock_Ranking#13]
"we normalize the price of each stock via dividing it by its maximum value throughout the entire 2013-2017 dataset"
in the paper, you used this way to normalize the value. however maximum value during the period is a future function
which can be known after the whole period.
this is very common mistake to use future function, did you try other normlizeing method?
thanks

**Comment by fulifeng** [comment]
Yes, before we use the maximum value throughout the entire 2013-2017 dataset, we have tried to use the maximum value during the training period. The results are quite close.
